src/controller/java/AndroidCallbacks.cpp

```diff
@@ -448,6 +448,10 @@ void ReportEventCallback::OnEventData(const app::EventHeader & aEventHeader, TLV
     readerForJavaTLV.Init(*apData);
     readerForJson.Init(*apData);
 
+    jlong eventNumber   = static_cast<jlong>(aEventHeader.mEventNumber);
+    jlong priorityLevel = static_cast<jint>(aEventHeader.mPriorityLevel);
+    jlong timestamp     = static_cast<jlong>(aEventHeader.mTimestamp.mValue);
+
```

Comment:
@joonhaengHeo please also add timestamp type, it could have either system timestamp or epoch timestamp.

Reply:
@joonhaengHeo are you in slack for matter project? maybe we can communicate in slack?